crates/sui-indexer-alt-jsonrpc/src/data.rs

```diff
@@ -28,11 +29,28 @@ pub(crate) async fn load_live(
         return Ok(None);
     }
 
-    let object = ctx
-        .kv_loader()
-        .load_one_object(object_id, latest_version.object_version as u64)
-        .await
-        .context("Failed to load latest object")?;
+    // Read from kv store and retry if the object is not found.
+    let mut object = None;
+    let config = &ctx.config().objects;
+    let mut interval = tokio::time::interval(Duration::from_millis(config.kv_retry_interval_ms));
+
+    for _ in 0..config.kv_retry_count {
```

Comment:
This should be an inclusive upperbound:

```suggestion
    for _ in 0..=config.kv_retry_count {
```

So that even if the system is configured not to retry at all, it issues at least one fetch.

Also, is `kv_retry_count` the right name for this? If it is, then we should also use this config for retrying fetching transactions, but we are not.

Either this should be `obj_retry_count`, or we should be using this retry count for fetching transaction contents as well.